lib/smartsheet/client.rb

```diff
@@ -33,6 +68,25 @@ class Client
                 :sheets, :sights, :templates, :token, :update_requests, :users, :webhooks,
                 :workspaces
 
+    # @param token [String] access token for the API; if nil or empty, uses environment variable
+    #   `SMARTSHEET_ACCESS_TOKEN`
+    # @param assume_user [String] the email address of the user to impersonate; only available for
```

Comment:
Does the ordering of these comments impact the order of the display? If so, it seems like we should have the more used options at the top (token, logger, json_output) and move assume_user to the bottom